build.gradle

```diff
@@ -80,6 +80,12 @@ configure(javaProjects) {
         }
     }
 
+    test {
+        testLogging {
+            events "passed", "skipped", "failed"
```

Comment:
If it's really only useful on travis can't we conditionally enable it by checking the existence system variable `CI`? So that we don't have to see a lot of stuff locally we don't really care about as we can always see the test reports locally.